data_structures/binary_tree/binary_tree_traversals.py

```diff
@@ -0,0 +1,149 @@
+class Node:
+    """
+    A Node has data variable and pointers toits left and right nodes.
+    """
+    def __init__(self,data):
+        self.left=None
+        self.right=None
+        self.data=data
+
+def preOrder(root):
+    """
+    PreOrder traversal: visit root node then its left subtree followed by right subtree.
+    """
+    if(root!=None):
+        print(root.data,end=" ")

```

Comment:
Algorithmic functions should not print() as discussed in CONTRIBUTING.md.  The caller can print() but not the algorithmic functions.